lib/spack/llnl/util/tty/__init__.py

```diff
@@ -156,6 +164,11 @@ def msg(message, *args, **kwargs):
 
 
 def info(message, *args, **kwargs):
+    if _osascript:
```

Comment:
Wrap this whole thing in a function so the code's not duplicated across all the `subprocess.call` invocations. And give the function a nice docstring.